free-programming-books-zh.md

```diff
@@ -24,6 +24,7 @@
   * [设计模式](#设计模式)
   * [软件开发方法](#软件开发方法)
   * [项目相关](#项目相关)
+  * [程序员杂谈](#程序员杂谈)
```

Comment:
Please insert this line and its section in alphabetical order. I can take care of it for you if you prefer, just tell me.